sdk/src/types/block/unlock/empty.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::block::Error;
+
+/// Used to maintain correct index relationship between addresses and signatures when unlocking a Multi Address where
```

Comment:
```suggestion
/// Used to maintain correct index relationship between addresses and signatures when unlocking a [`MultiAddress`](crate::types::block::address::MultiAddress) where
```
not sure if fmt is happy with the suggestion